src/context.rs

```diff
@@ -356,9 +418,35 @@ mod tests {
         }
 
         #[test]
+        fn it_can_call_given_inside_describe_body() {
+            describe("A Root", |ctx| ctx.given("nested describe", |_ctx| {}));
+        }
+
+        #[test]
+        fn it_can_call_when_inside_describe_body() {
+            describe("A Root", |ctx| ctx.when("nested describe", |_ctx| {}));
+        }
```

Comment:
missing tests for `context` and `specify`. You may want a macro, like:

```rust
macro_rules! test_describe_alias {
  ($alias: ident) => {
    #[test]
    fn describe_has_alias_$alias() {
      describe("A Root", |ctx| ctx.$alias("nested describe", |_ctx| {}));
    }
  }
}
```

(made from memory, without testing, it may not compile...)